apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -262,6 +307,12 @@ export const LockfileEntryDetailsView = (): JSX.Element | ReactNull => {
                   <div>
                     <Text type="p">Entry ID: {referrer.rawEntryId}</Text>
                   </div>
+                  <div>
+                    <Text type="p">
+                      {'Dependency version: '}
+                      {getDependencyInfo(referrer.rawEntryId, selectedEntry.entryPackageName)?.version}
+                    </Text>
+                  </div>
```

Comment:
This increases the vertical size of each entry, which may require extra scrolling to view the list.

Could we make it more compact somehow?  Maybe something like this:

> ### Direct Referrers
>
> **Name: @babel/core 7.12.9 (^7.10.4)**
> Entry ID: /@babel/core/7.12.9

@william2958 what do you think?

Reply:
I think this adjustment will be more intuitive, I can change it to what you said first.

Reply:
We can move the second text a bit closer to the first (put them in the same div) to save some space:

```
<div>
  <Text type="p">Entry ID: {referrer.rawEntryId}</Text>
  <Text type="p">
    {'Dependency version: '}
    {getDependencyInfo(referrer.rawEntryId, selectedEntry.entryPackageName)?.version}
  </Text>
</div>
```

Reply:
done